framework/doc/content/source/auxkernels/PenetrationAux.md

```diff
@@ -2,6 +2,10 @@
 
 Auxililary Kernel for computing several geometry related quantities between two different bodies in or near contact.
 
+## Gap offset parameters
+
+Gap offset can be provided to offset the gap distance between in or near contact bodies. It can be either `secondary_gap_offset` (gap offset from secondary side) or `mapped_primary_gap_offset` (gap offset from primary side but mapped to secondary side). However, the offsetted gap is treated as rigid region without deformation and temperature gradient.
```

Comment:
I'd re-word the last sentence:
Use of these gap offset parameters treats the surfaces as if they were virtually extended by the specified amount, so that the surfaces are treated as if they are closer than they actually are. There is no deformation or resistance to heat transfer within the material in this gap offset region.